src/lib/core/CoreLayer.am

```diff
@@ -54,4 +55,11 @@ CHIP_BUILD_CORE_LAYER_HEADER_FILES                        = \
     @top_builddir@/src/lib/core/Optional.h                  \
     @top_builddir@/src/lib/core/ReferenceCounted.h          \
     @top_builddir@/src/lib/core/CHIPVendorIdentifiers.hpp   \
+    @top_builddir@/src/lib/core/CHIPBaseAttribute.h   \
```

Comment:
recommend moving this to a datamodel directory (to match namespace)

Reply:
done